<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_blockCipher">
        <property name="text">
         <string>Block Cipher</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_hash">
        <property name="text">
         <string>Hash</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_MAC">
        <property name="text">
         <string>MAC</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page_home">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>180</y>
          <width>151</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>home page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_blockCipher">
       <property name="minimumSize">
        <size>
         <width>631</width>
         <height>0</height>
        </size>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>201</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>block cipher page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_hash">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>160</y>
          <width>211</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>hash page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_MAC">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>150</y>
          <width>191</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>MAC page</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menudd">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menudd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
